tfe/data_source_organization_tags.go

```diff
@@ -0,0 +1,81 @@
+package tfe
+
+import (
+	"fmt"
+	"strconv"
+
+	tfe "github.com/hashicorp/go-tfe"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+func dataSourceTFEOrganizationTags() *schema.Resource {
+	return &schema.Resource{
+		Read: dataSourceTFEOrganizationTagsRead,
+
+		Schema: map[string]*schema.Schema{
+			"organization": {
+				Type:     schema.TypeString,
+				Optional: true,
+			},
+			"tags": {
+				Type:     schema.TypeList,
+				Computed: true,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"name": {
+							Type:     schema.TypeString,
+							Computed: true,
+						},
+
+						"id": {
+							Type:     schema.TypeString,
+							Computed: true,
+						},
+
+						"workspace_count": {
+							Type:     schema.TypeString,
```

Comment:
I think this should be a `schema.TypeInt`, to simplify using it as a number from within Terraform configs.